markdown/parsing/ll-advanced.md

```diff
@@ -273,7 +274,11 @@ zurücksetzt, werden alle `*_memo` ungültig und müssen ebenfalls zurückgesetz
 :::
 
 [[Anmerkung Anpassung `consume()`]{.bsp}]{.slides}
+REM -->
 
+<!-- ADD
+Der restliche Inhalt soll Teil von Vorlesung 8 (ehem. 4.2) "Attributierte Grammatiken werden"
+ADD -->
```

Comment:
Ich wollte das jetzt noch nicht direkt verschieben, um die Gefahr von Konflikten möglichst gering zu halten. Reicht das so aus? Doof daran ist auf jeden Fall, dass die Inhalte nicht mehr im Struktur PDF auftauchen (vmtl weil der Header auskommentiert ist; was ja aber so gedacht ist, weil das Kapitel an sich rausfliegen soll).

Reply:
Hmmm. Wenn Du in dem einen Branch die Inhalte neu (andere Datei) hinzufügst und in dem anderen Branch die alten Inhalte löschst, dann sollte das doch konfliktfrei gehen, oder? Das würde ich schon machen, sonst geht es hinterher komplett durcheinander.